src/adapter/breakpoints.ts

```diff
@@ -585,6 +585,11 @@ export class BreakpointManager {
 
     this._totalBreakpointsCount += result.new.length;
 
+    if (this._thread && this._totalBreakpointsCount === 0 && this._sourceMapHandlerWasUpdated) {
+      this._thread.setScriptSourceMapHandler(false);
+      this._sourceMapHandlerWasUpdated = false;
```

Comment:
Let's rename `_sourceMapHandlerWasUpdated` to `_sourceMapHandlerInstalled` to better reflect what it's doing.

Reply:
should _updateSourceMapHandler also be renamed to _installSourceMapHandler? It may also make sense to have an _uninstallSourceMapHandler, but I had been trying to make my original change as minimal as possible. I didn't really like that I was manually messing with a flag that seems to be owned by the update method, but it was awkward to try to make the update method "figure out" what do do in most cases, but then guarantee that it will uninstall the source map handler in the case I added.

Reply:
I'm good with _installSourceMapHandler/_uninstallSourceMapHandler

Reply:
I updated the pull request with those changes